cmd/build-init/main.go

```diff
@@ -104,15 +106,22 @@ func main() {
 		}
 	}
 
-	err = dockercreds.VerifyWriteAccess(creds, *imageTag)
+	logger.Println("Loading cluster credential helpers")
+	k8sNodeKeychain, err := k8schain.NewNoClient(context.Background())
+	if err != nil {
+		logger.Fatal(err)
+	}
+	keychain := authn.NewMultiKeychain(creds, k8sNodeKeychain)
+
+	err = dockercreds.VerifyWriteAccess(keychain, *imageTag)
```

Comment:
Super minor but, would it make sense to inline keychain to call out the limited scope of this keychain?

Reply:
What do you mean by "call out"?

Reply:
Make it limited scope clear. This keychain is only used in this location but, that is unclear.